.eslintrc.js

```diff
@@ -1,51 +1,191 @@
+/* eslint-env node */
```

Comment:
@jgroth There was a contradiction in the rules for how eslint and prettier wanted `switch` statements to be formatted, so rather than trying to find and fix that specific discrepancy, I just copied the config from lime-elements, and installed the missing node modules… I also disabled `no-magic-numbers` and `no-shadow`, since I didn't want to fix those, and I'm guessing those might not be your favourite rules anyway 😄

Reply:
Errr… the lint-check is failing 🤦‍♂️

Reply:
But it's passing locally… what the heck!? 🧐

Reply:
_Finally!_

![image](https://user-images.githubusercontent.com/154725/119134303-464aa300-ba3d-11eb-9a3a-ce156c009f3f.png)